packages/webapp/pages/join.tsx

```diff
@@ -31,6 +33,10 @@ const Page = ({
     referringUser,
   });
 
+  useEffect(() => {
+    document.body.classList.add('hidden-scrollbar');
```

Comment:
Just for safety, we should remove this class on unmount.

Reply:
Yep agreed, let's add the removal as well.
Tested and seems to be fine for this page also on mobile.